README.md

```diff
@@ -24,7 +24,25 @@ A larger set of interesting patterns might include:
 * `^spew\.Dump$` -- forbid dumping detailed data to stdout
 * `^fmt\.Errorf(# please use github\.com/pkg/errors)?$` -- forbid Errorf, with a custom message
 
-Note that the linter has no knowledge of what packages were actually imported, so aliased imports will match these patterns.
+By default, patterns get matched against the actual expression as it appears in
```

Comment:
Note that I am not changing the default patterns because I didn't want to cause regressions. Some (all?) of the default patterns are probably better expressed as matches against the package, but then they wouldn't work anymore in the stand-alone forbidigo binary because there the `types.Info` pointer is nil.

Reply:
I meant to say that I think if we can get types.Info populated then we could update the defaults, but I'm happy to wait until subsequent PR.

Reply:
I am doing it in this PR with a separate commit. That commit also shows how the default behavior changes.